setup.py

```diff
@@ -64,6 +72,7 @@ def find_version(*file_paths):
     name="tarantool",
     packages=["tarantool"],
     package_dir={"tarantool": os.path.join("tarantool")},
```

Comment:
https://docs.python.org/3/distutils/setupscript.html#listing-whole-packages
```suggestion
    package_dir={"tarantool": "tarantool"},
```

Reply:
Thanks, tried to use it